code/script.js

```diff
@@ -34,18 +35,378 @@ const showMessage = (message, sender) => {
+    })
+  } 
+
+//bot asks for feedback on the shop when ordering plain donuts
+const finalQuePlain = (selection) => {
+  if (selection === "delivery1") { 
+    showMessage("Thank you for ordering our plain donuts! We will deliver ASAP! Before you go, do you agree that our donut shop is great?", 'bot');
+    setTimeout(() => yayOrNay(), 900)
+  }
+
+  else if (selection === "delivery2", "delivery3") {
+    showMessage("Thank you for ordering our plain donuts, we will deliver according to your selected time frame. Before you go though, please let us know if you think our donut shop is amazing!", 'bot')
+    setTimeout(() => yayOrNay(), 900)
+  }
+} 
+
+//bot asks for feedback when ordering filled donuts
+const finalQueFilled = (selection) => {
+  if (selection === "delfill1") { 
+    showMessage("Thank you for ordering our filled donuts! We will deliver ASAP! Before you go, do you agree that our donut shop is great?", 'bot');
+    setTimeout(() => yayOrNay(), 900)
+  }
+
+  else if (selection === "delfill2", "delfill3") {
+    showMessage("Thank you for ordering our filled donuts, we will deliver according to your selected time frame. Before you go though, please let us know if you think our donut shop is amazing!", 'bot')
+    setTimeout(() => yayOrNay(), 900)
+  }
+} 
+//Feedback on the donut shop - options
+//Saying that it sucks should render a pop-up answer about answer invalid, as well as a message
+const yayOrNay = () => {
+  inputWrapper.innerHTML = `
+    <button id="yay">Of course, it is the best!</button>
+    <button id="nay">No, it really sucks!</button>
+    `
+
+document
+  .getElementById("yay")
+  .addEventListener("click", () => {
+    showMessage("I love it!", 'user')
+    inputWrapper.innerHTML = ''
+    setTimeout(() => trueOrFalse("yay"), 1000)
+  })
+
+document
+  .getElementById("nay")
+  .addEventListener("click", () => {
+    showMessage("I'm a banana!", 'user')
+    inputWrapper.innerHTML = ''
+    setTimeout(() => trueOrFalse("nay"), 1000)
+  })
+} 
+
+const trueOrFalse = (selection) => { 
+  if (selection === "yay") {
+    showMessage("Welcome back anytime!💕", 'bot')
+    setTimeout(() => inputWrapper.innerHTML= `
+    <p>Have a lovely day!</p>`, 1000)
+  }
+
+  else if  (selection === "nay") {
+    alert("Sorry, your selected option is invalid.. 🙈🍩🙈")
+    setTimeout(() => inputWrapper.innerHTML= `
+    <p>Have a lovely day!</p>`, 1000)
+  }
 }
 
 // Set up your eventlisteners here
+form.addEventListener('submit', (event) => {
+event.preventDefault();
+})
+
+
 
 // When website loaded, chatbot asks first question.
 // normally we would invoke a function like this:
 // greeting()
 // But if we want to add a little delay to it, we can wrap it in a setTimeout:
 // setTimeout(functionName, timeToWaitInMilliSeconds)
 // This means the greeting function will be called one second after the website is loaded.
-setTimeout(greetUser, 1000);
+setTimeout(greetUserOrder, 900);
```

Comment:
Minor detail, but just for reading-purposes, I think this should placed after the greeting function.